flax/nn/base.py

```diff
@@ -200,7 +202,7 @@ class _ModuleMeta(abc.ABCMeta):
 
   def __init__(cls, name, bases, attrs):
     super(_ModuleMeta, cls).__init__(name, bases, attrs)
-    apply_fn = cls.apply
+    apply_fn = cls.apply  # pytype: disable=attribute-error
```

Comment:
Would this error go away if we type `cls` with something like `Callable[..., Any]`? Then it should know the `appl` function exists.

Reply:
Pytype refuses a type hint on the self argument (which is cls in this case) :(